js/infobar.js

```diff
@@ -0,0 +1,466 @@
+'use strict';
+
+var $infoBar = {};
+var $message = {};
+var $acceptButton = {};
+var $cancelButton = {};
+var bars = {};
+var opened = false;
+
+var bar = {};
+var json = {};
+var lang = '';
+var availableLangs = {};
+var acceptLangs = [];
```

Comment:
I agree, I was leaning towards that, perhaps even using a config object like suggested by Nicholas Zakas. ex.

```
var config = {
    $infoBar = {},
    $message = {},
    $acceptButon = {},
    ...
};
```
Also agree on wrapping the code up, because I *really* do not like polluting the global namespace like it is here. Only reason I have not yet done so, is I was kinda concerned with unit testing but, I can of course expose the `InfoBar` object to global scope and thus enable unit testing.